pipedrive/client-php
# Fix due date for update activity mapping

## Problem
`updateEditAnActivity` did not take into account `dueDate` as autocreated mapping was incorrect.

## How to test

```php
// should update a due date for an existing activity 
$query['dueDate'] = new DateTime('2021-04-02');
$query['id'] = '<an existing activity id>';

$client->getActivities()->updateEditAnActivity($query);   
```

Related PRs:
https://github.com/pipedrive/client-php/pull/28
